.evg.yml

```diff
@@ -47,6 +47,16 @@ buildvariants:
       - name: "test-unit"
       - name: "test-integration"
 
+  - name: ubuntu2204-arm64-jdk-8
```

Comment:
Should this be amazon linux?

Reply:
I converted the PR to draft to work on adding amazon linux to address the latest comments.